[2021-07-17 14:00:45,541] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.bash_task3 2021-07-17T14:00:06.054907+00:00 [queued]>
[2021-07-17 14:00:45,549] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.bash_task3 2021-07-17T14:00:06.054907+00:00 [queued]>
[2021-07-17 14:00:45,549] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-17 14:00:45,549] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-17 14:00:45,549] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-17 14:00:45,560] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): bash_task3> on 2021-07-17T14:00:06.054907+00:00
[2021-07-17 14:00:45,563] {standard_task_runner.py:52} INFO - Started process 9047 to run task
[2021-07-17 14:00:45,565] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'parallel_dag', 'bash_task3', '2021-07-17T14:00:06.054907+00:00', '--job-id', '840', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpnqdnvu8l', '--error-file', '/tmp/tmpttqqu2gt']
[2021-07-17 14:00:45,565] {standard_task_runner.py:77} INFO - Job 840: Subtask bash_task3
[2021-07-17 14:00:45,589] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.bash_task3 2021-07-17T14:00:06.054907+00:00 [running]> on host 120364900fc4
[2021-07-17 14:00:45,652] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=bash_task3
AIRFLOW_CTX_EXECUTION_DATE=2021-07-17T14:00:06.054907+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-17T14:00:06.054907+00:00
[2021-07-17 14:00:45,652] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-17 14:00:45,653] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-07-17 14:00:45,659] {subprocess.py:75} INFO - Output:
[2021-07-17 14:00:48,660] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-17 14:00:48,721] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=bash_task3, execution_date=20210717T140006, start_date=20210717T140045, end_date=20210717T140048
[2021-07-17 14:00:48,753] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-17 14:00:48,784] {local_task_job.py:151} INFO - Task exited with return code 0
